casanovo/denovo/model.py

```
@@ -1213,14 +1215,43 @@ def predict_step(
             pred.protein = self.protein_database.get_associated_protein(
                 pred.sequence
             )
+            calc_mass = self.protein_database.db_peptides.loc[
+                pred.sequence, "calc_mass"
+            ]
+            pred.calc_mz = float(calc_mass) / pred.charge + PROTON
 
         return predictions
 
+    def on_predict_batch_end(
+        self, outputs: List[psm.PepSpecMatch], *args
```

Ruff (0.15.7)

[warning] Missing type annotation for `*args`

(ANN002)